Legacy/bonej/src/main/java/org/bonej/plugins/ParticleCounter.java

```diff
@@ -300,6 +308,20 @@ public void run(final String arg) {
 						rt.addValue("vZ2", E.getV().get(2, 2));
 					}
 				}
+				if (doSkeletons) {
```

Comment:
`if (skeletonResult) != null`?

Reply:
Prefer this way, it's clearer communication of intent. `skeletonResult` is only not `null` if `doSkeletons = true`